Formula/beets.rb

```diff
@@ -0,0 +1,150 @@
+class Beets < Formula
+  include Language::Python::Virtualenv
+  desc "Music library manager and MusicBrainz tagger"
+  homepage "http://beets.io/"
+  url "https://github.com/beetbox/beets/releases/download/v1.4.6/beets-1.4.6.tar.gz"
+  sha256 "62079b2338799a64e7816096c5fae3b8909fb139e4d481ec3255336e67765b50"
+
+  option "without-fetchart", "don't install python dependencies for the fetchart plugin"
```

Comment:
Making all of these a hard default, except for perhaps `ffmpeg`, seems completely reasonable.

Reply:
Sure! I considered this, but decided replicating the official pip-based install method seemed more useful - especially as, for instance, the `absubmit` plugin also has a native dependency, which will eventually make its' way into Homebrew. 

I guess making these default would make bottling the formula easier. 

I'm tempted to roll all the python deps into the formula, and only have flags for the plugins which have native dependencies?

The only decision to make _then_ would be whether to have a flag per native dep, or to group them by beets plugin: right now there are not plugins distributed with beets which have multiple native dependencies, but there's no reason there _won't_ be, and I don't want to make a breaking change to the formula flags down the line. 

(I made an exception for `ffmpeg`, because... it's `ffmpeg` and it's a dep of beets directly, I feel like it should be a flag of its' own.)